src/sonic-yang-models/yang-models/sonic-vlan.yang

```diff
@@ -80,6 +80,13 @@ module sonic-vlan {
 					}
                                         default "0";
 				}
+				leaf mpls {
+					description "Enable/disable MPLS routing for the vlan interface";
+					type enumeration {
+						enum enable;
+						enum disable;
```

Comment:
Please define typedef for enable/disable and use it here.

Reply:
@venkatmahalingam There seems to be some inconsistency with regards to typedefs in sonic-yang-models.  I see instances of enable/disable enum in sonic-device_metadata.yang and sonic-flex_counter.yang without a generic typedef being referenced.  Also, I see that sonic-types.yang now requires a revision.  I would suggest that a new revision of sonic-types.yang that incorporates a generic enable/disable typedef should be outside of the scope of this PR and done as a general cleanup of the yang_models to avoid churn on yang revisions.

Reply:
why can't we add typedef for enable/disable in the sonic-types.yang and use it in this YANG? we can correct typedef in the other modules later?

Reply:
@venkatmahalingam Presumably any addition to sonic-types.yang requires a new revision for that file.  It makes more sense to bundle multiple changes to sonic-types.yang together to avoid revision churn.  This should be done as general cleanup prior to new release and should not be included in the scope of this PR.